sim/core/cast.go

```diff
@@ -204,10 +199,12 @@ func (spell *Spell) makeCastFunc(config CastConfig) CastSuccessFunc {
 			spell.Unit.SetGCDTimer(sim, sim.CurrentTime+effectiveTime)
 		}
 
+		// TODO: Fix with removal of ChannelTime?
 		if (spell.CurCast.CastTime > 0 || spell.CurCast.ChannelTime > 0) && spell.Unit.Moving {
```

Comment:
Unsure if we should just remove these completely? These are unique to `sod-fork`